.gitignore

```diff
@@ -5,9 +5,6 @@ logs/
 fabfile.py
```

Comment:
fabfile.py ?
NOTES.md ?

Reply:
no idea.  do you have anymore  to request while i'm in that file :-)

Reply:
logs/ ?

Reply:
hmm, i wonder if logs might be created in a diagnosing mode of any of our tools.